plat/arm/common/arm_bl2_setup.c

```diff
@@ -37,7 +37,7 @@ CASSERT(BL2_BASE >= ARM_TB_FW_CONFIG_LIMIT, assert_bl2_base_overflows);
 
 #if LOAD_IMAGE_V2
 
-#pragma weak bl2_plat_handle_post_image_load
+#pragma weak arm_bl2_plat_handle_post_image_load
```

Comment:
This function should not be weak - we do not expect it to be overridden by a specific platform.